README.md

```diff
@@ -44,10 +45,20 @@ Add this to your init.el:
 (autoload 'rust-mode "rust-mode" nil t)
 ```
 
+# Indentation
+
+The Rust style guide recommends spaces for indentation; to follow the
+recommendation add this to your init.el:
+
+```elisp
+(add-hook 'rust-mode-hook
+	  (lambda () (setq indent-tabs-mode nil)))
```

Comment:
is that a tab?

Reply:
That is embarrassing considering the point of the PR :)

I'll fix that.